pgamit/pyETM.py

```diff
@@ -2438,17 +2505,21 @@ def plot_hist(self, pngfile=None, fileio=None):
             ax.set_xlabel(LABEL('U residuals') + ' [mm]')
             ax.set_title(LABEL('histogram plot') + ' ' + LABEL('up'))
 
-            # residuals = np.sqrt(np.square(L[0]) + np.square(L[1]) + np.square(L[2])) - \
-            #            np.sqrt(np.square(np.dot(self.A, self.C[0])) + np.square(np.dot(self.A, self.C[1])) +
-            #                    np.square(np.dot(self.A, self.C[2])))
+            # residuals = np.sqrt(np.square(L[0]) + np.square(L[1]) + \
+            #   np.square(L[2])) - \
+            #   np.sqrt(np.square(np.dot(self.A, self.C[0])) + \
+            #   np.square(np.dot(self.A, self.C[1])) + \
+            #   np.square(np.dot(self.A, self.C[2])))
 
             # (mu, sigma) = norm.fit(residuals)
 
-            # n, bins, patches = plt.hist(residuals, 200, normed=1, alpha=0.75, facecolor='blue')
+            # n, bins, patches = plt.hist(residuals, 200, normed=1,
+            #                             alpha=0.75, facecolor='blue')
 
             # y = mlab.normpdf(bins, mu, sigma)
             # plt.plot(bins, y, 'r--', linewidth=2)
-            # plt.title(r'$\mathrm{Histogram\ of\ residuals (mm):}\ \mu=%.3f,\ \sigma=%.3f$' % (mu*1000, sigma*1000))
+            # plt.title(r'$\mathrm{Histogram\ of\ residuals (mm):}\
+            #   \mu=%.3f,\ \sigma=%.3f$' % (mu*1000, sigma*1000))
```

Comment:
@demiangomez can we start deleting some of the commented out functions? We'll still have the git history and can recover the code from there if we decide it's needed again down the road